pandas/tests/frame/indexing/test_where.py

```diff
@@ -189,6 +189,8 @@ def _check_set(df, cond, check_dtypes=True):
             with pytest.raises(TypeError, match=msg):
                 df > 0
             return
+        if df is mixed_int_frame:
```

Comment:
can this be reached?

Reply:
yup: `pandas/tests/frame/indexing/test_where.py::TestDataFrameIndexingWhere::test_where_set[mixed_int]`